LoginPlugin/iOS/README.md

```diff
@@ -0,0 +1,147 @@
+     */
+    public required init(configurationJSON: NSDictionary?) {
+        super.init()
+        self.loginManager = LoginManager()
+        self.configurationJSON = configurationJSON
+    }
+    
+    /**
+     This method is being called in order to start login process.
+     The completion should always be called when the process is done - no matter what is the result.
+     */
+    public func login(_ additionalParameters: [String : Any]?, completion: @escaping ((ZPLoginOperationStatus) -> Void)) {
+        if let loginManager = loginManager {
+            loginManager.logIn(readPermissions:[ .publicProfile ], viewController: nil) { loginResult in
+                switch loginResult {
+                case .failed(let error):
+                    print(error)
+                    completion(.failed)
+                case .cancelled:
+                    print("User cancelled login.")
+                    completion(.cancelled)
+                case .success( _, _, let accessToken):
+                    print("Logged in!")
+                    if accessToken.authenticationToken.isEmptyOrWhitespace() == false {
+                        self.setUserToken(token: accessToken.authenticationToken)
+                        self.setUserExpirationDate(token: accessToken.expirationDate)
+                        completion(.completedSuccessfully)
+                    } else {
+                        completion(.failed)
+                    }
+                }
+            }
+        } else {
+            completion(.failed)
+        }
+    }
+    
+    /**
+     This method is being called in order to start logout process.
+     The completion should always be called when the process is done - no matter what is the result.
+    */
+    public func logout(_ completion: @escaping ((ZPLoginOperationStatus) -> Void)) {
+        if let loginManager = loginManager {
+            loginManager.logOut()
+            completion(.completedSuccessfully)
+        } else {
+            completion(.failed)
+        }
+    }
+    
+    /**
+     This methood is called in order to verify if we need to start a login flow
+     Returns bool value indicating if the user is already verified if not  we start the login proccess
+     in this example we check if the login token exists and if it is valid
+    */
+    public func isAuthenticated() -> Bool {
+        var retVal = false
+        if let expirationDate = self.getUserExpirationDate(),
+            self.getUserToken().isEmpty == false {
+            let currentDate = Date()
+            if expirationDate > currentDate || expirationDate == currentDate {
+                retVal = true
+            }
+        }
+        return retVal
+    }
+    
+	/**
+     This methood is called in order to verify if authorization flow is in process
+    */
+    public func isPerformingAuthorizationFlow() -> Bool {
+        return self.isPerformingAuthorizationFlow()
+    }
+    ```
```

Comment:
you need to remove this space, otherwise, it all will be one big code view.

Reply:
It is seperated

Reply:
![screen shot 2018-05-22 at 15 25 19](https://user-images.githubusercontent.com/4838424/40362178-76eea600-5dd4-11e8-88be-a4957c16c020.png)

Reply:
well, @RoiKed ... the GitHub rendering is the one that counts...